Examples/socketio-python/README.md

```diff
@@ -0,0 +1,79 @@
+## Python Socketio Example
+
+This example illustrates one way to utilize socketio for data transmittal.
+
+There are 3 files:
+
+`client_logger.py`
+- Socketio client that listens for event "log_telemetry" and logs
+the data packet to console
+
+`client_telemetry.py`
+- Socketio client that generates telem (simulates receiving from the TBM) and emits a "log_telemetry" event containing the data packet
+
+`server.py`
+- Standard socketio server, handles event routing between clients  
+
+The primary difference between this implementation and the actual implementation is the packet type and the server web app. We are looking to use protobuf, while this initial example uses json for simplicity. In the future, this example can be duplicated using protobuf. Also, this example uses an eventlet web app - in production socketio will be served by our angular application. 
+
+### Running the Example
+1. Ensure python environment is activated 
+2. `pip install -r requirements.txt`
+3. `python3 server.py`
+4. `python3 client_telemetry.py`
+5. `python3 client_logger.py`
+
+## Socketio
+
+### Server
+Defining an event
+```
+@sio.event
+def my_event(sid, data):
+    pass
+
+# OR
+
+@sio.on('my event name')
+def another_event(sid, data):
+    pass
+```
```

Comment:
In flask-socketio, you have to use `@sio.on()` though right?

Reply:
True - resolved